bundled-libunwind/test.json

```diff
@@ -6,6 +6,10 @@
   "versionSpecific": false,
   "type": "bash",
   "cleanup": true,
+  "skipWhen": [
+    "os=fedora,version=6", // .NET 6 on Fedora uses system libunwind.
+    "os=fedora,version=7"  // .NET 7 on Fedora uses system libunwind.
+  ],
```

Comment:
This is now duplicating `ignoredRIDs` below. That list already contains Fedora so it would skip this test on all versions of Fedora already.

Reply:
It seems that .NET 8 behavior is different? See https://github.com/redhat-developer/dotnet-regular-tests/pull/252#discussion_r1228021274.